CHANGELOG.md

```diff
@@ -6,7 +6,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 ## [Unreleased]
 
+### Changed
+
+- Various styling enhancements to typography and colours.
```

Comment:
Why is this here in this MR?

Reply:
I forgot to add this stuff to the changelog last time in #188, so just doing it this time around :)